worldengine/cli/main.py

```diff
@@ -391,8 +391,8 @@ def main():
     if args.seed is not None:
         seed = int(args.seed)
     else:
-        seed = random.randint(0, 65535)#RNG initialization is done automatically
-    random.seed(seed)
+        seed = numpy.random.randint(0, 999999)  # RNG initialization is done automatically
```

Comment:
@ftomassetti and I talked about this. It isn't a problem in Python, but in Platec. I thought its seed was a 16-bit unsigned number. Not sure if this still the case. Honestly I don't like 99999 either... I would prefer max_int.